test/testscript.go

```diff
@@ -32,27 +32,70 @@ func NewProjectTest(t *testing.T, projectDir string) *ProjectTest {
 	}
 }
 
+func copyFile(srcPath string, dstPath string, mode os.FileMode) error {
```

Comment:
maybe a bit much to move these into separate funcs since they are so small but they were shared between CopyFile and the new CopyDir